presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -177,7 +177,9 @@ class EventViewModel @Inject constructor(
             is EventScreenEvent.OnAttendeeAdded -> {
                 _eventScreenState.update { eventScreenState ->
                     eventScreenState.copy(
-                        attendees = eventScreenState.attendees + eventScreenEvent.attendee
+                        attendees = eventScreenState.attendees + eventScreenEvent.attendee.copy(isGoing = true),
+                        filteredVisitorsGoing = eventScreenState.attendees.filter { it.isGoing },
+                        filteredVisitorsNotGoing = eventScreenState.attendees.filter { !it.isGoing }
                     )
                 }
             }
```

Comment:
You are going to call this only in the ViewModel. No need for an event. You can just have a function for it, and call it on the verifyVisitorEmail method